exercises/word-count/canonical-data.json

```diff
@@ -125,6 +125,26 @@
         "cry": 1
       }
     },
+    {
+      "uuid": "4ff6c7d7-fcfc-43ef-b8e7-34ff1837a2d3",
+      "reimplements": "4185a902-bdb0-4074-864c-f416e42a0f19",
+      "description": "with apostrophes",
+      "comments": ["Added a word with multiple letters after apostrophe"],
+      "property": "countWords",
+      "input": {
+        "sentence": "'First: don't laugh. Then: don't cry. You're getting it.'"
+      },
+      "expected": {
+        "first": 1,
+        "don't": 2,
+        "laugh": 1,
+        "then": 1,
+        "cry": 1,
+        "you're": 1,
```

Comment:
I notice that the "don't" has a value of 2, while "you're" has a value of 1.  Should it not be a 2 as well?

Reply:
There's only one instance of "you're" in the input text. Or do you mean we should use an input that includes "you're" multiple times?

Reply:
Nope, sorry did not associate it directly with the inputs as a word count occurrence.  Thought perhaps it was a contraction being counted as the two words it involves.